storagemarket/provider.go

```diff
@@ -297,6 +298,7 @@ func (p *Provider) ExecuteDeal(ctx context.Context, dp *types.DealParams, client
 		DealDataRoot:       dp.DealDataRoot,
 		Transfer:           dp.Transfer,
 		IsOffline:          dp.IsOffline,
+		CleanupData:        !dp.IsOffline,
```

Comment:
Can you clarify why this is the inverse of IsOffline? I want to ensure I'm clear on accepting network deals and what cleanup eventually looks like based on this value.

Reply:
CleanupData is the new flag we've added to the state to indicate whether to remove the downloaded / imported file after it's been handed off to the sealing subsystem.

By default the flag is
- true for online deals (remove data that was downloaded after it's been handed off to the sealing subsystem)
- false for offline deals (don't remove data that was imported)

When the user runs `boostd import-data` they can set a flag to explicitly tell boost to remove the imported data after it's been handed off to the sealing subsystem. The CLI command will set `CleanupData` to true.